.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Space CI
+on: 
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main"]
```

Comment:
It's probably best to have this be for pull requests against all branches and not only `main`. If we open a pull request for a branch that is not `main`, [this workflow won't run](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-the-head-or-base-branch-of-a-pull-request).

You can simple change the `branches: ["main"]` line of the `pull_request` section to `branches: '*'`

Reply:
Perfect, thank you!